solvebio/resource/object.py

```diff
@@ -491,7 +491,7 @@ def lowercase(x):
 
         return lowercase(tag) in map(lowercase, self.tags)
 
-    def tag(self, tags, remove=False, dry_run=False, apply_save=False):
+    def tag(self, tags, remove=False, dry_run=False, apply_save=True):
```

Comment:
This is not a new issue, but what happens if somebody puts in a string? o.tag("something"), will it add the following tags: [s, o, m, ...]? We might want to add a check to case the input to a list.